lib/dagDisplay.ml

```diff
@@ -35,29 +35,55 @@ let image_normal_txt conf base p fname width height =
   in
   {|<a href="|}
   ^ (commd conf : Adef.escaped_string :> string)
-  ^ {|m=IM&|} ^ (b : Adef.escaped_string :> string) ^ {|&k=/|} ^ k ^ {|">|}
-  ^ (r : Adef.escaped_string :> string) ^ "</a>"
+  ^ {|m=IM&|}
+  ^ (b : Adef.escaped_string :> string)
+  ^ {|&k=/|}
+  ^ k
+  ^ {|">|}
+  ^ (r : Adef.escaped_string :> string)
+  ^ "</a>"
   |> Adef.safe
 
 let image_url_txt conf url_p url height : Adef.safe_string =
   let image_txt = Utf8.capitalize_fst (transl_nth conf "image/images" 0) in
-  Adef.safe @@
-  {|<a href="|} ^ (url_p : Adef.escaped_string :> string) ^ {|"><img src="|} ^ (url : Adef.escaped_string :> string)
-  ^ {|" alt="|} ^ image_txt ^ {|" title="|} ^ image_txt ^ {|" style="|}
-  ^ (if height = 0 then "" else "max-height:" ^ string_of_int height ^ "px;")
-  ^ {|">|} ^ {|</a>|}
+  {|<a href="|}
+  ^ (url_p : Adef.escaped_string :> string)
+  ^ {|"><img src="|}
+  ^ (url : Adef.escaped_string :> string)
+  ^ {|" alt="|}
+  ^ image_txt
+  ^ {|" title="|}
+  ^ image_txt
+  ^ {|" style="|}
+  ^ (if height = 0 then "" else "max-height:"
+  ^ string_of_int height ^ "px;")
+  ^ {|">|}
+  ^ {|</a>|}
```

Comment:
to make long concatenation like that more readable you can use Format: 
`Format.sprintf {|<a href="%s"><img src="%s" alt"%s" ....|}
                               (url_p : Adef.escaped_string :> string) (url : Adef.escaped_string :> string) image_txt ...`

Reply:
OK for `image_url_txt` and `image_url_txt_with_size`.
Tried to do the same for `image_normal_txt`. Please double check